tests/src/test/scala/org/apache/openwhisk/core/entity/test/ExecManifestTests.scala

```diff
@@ -319,7 +319,7 @@ class ExecManifestTests extends FlatSpec with WskActorSystem with StreamLogging
                  |""".stripMargin.parseJson.asJsObject
 
     val js6 = RuntimeManifest(
-      "nodejs:10",
+      "nodejs:6",
```

Comment:
I think it would be better to drop tests relevant to nodejs:6 if possible.

Reply:
Actually, nodejs6 is just a name here, and on the other hand, runtime nodejs:6 is already deprecated in runtimes.json, so i agree with `drop tests relevant to nodejs:6`

In current json doc, `nodejs6 kind` includes 1 stem type(128 M), `nodejs8 kind` includes 2 stem type(128 M and 256 M), so i think in that doc, we need to keep 1 stem type for nodejs as well.

And as above said ` nodejs:6 is already deprecated` and it is just a name, so i changed `nodejs6` to `nodejs10`